### PR TITLE
Update onboard-windows-multi-session-device.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a limitation that Only Microsoft Edge is supported for web filtering in windows 10 multisession and network protection doesn't work in inspecting 3rd party browsers

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7674

File: microsoft-365/security/defender-endpoint/onboard-windows-multi-session-device.md [hunk 1]
```diff
@@ -153,6 +153,10 @@ Also, if you're using FSlogix user profiles, we recommend you exclude the follow
 Note on licensing: When using Windows Enterprise multi-session, depending on your requirements, you can choose to either have all users licensed through Microsoft Defender for Endpoint (per user), Windows Enterprise E5, Microsoft 365 Security, or Microsoft 365 E5, or have the VM licensed through Microsoft Defender for Cloud.
 Licensing requirements for Microsoft Defender for Endpoint can be found at: [Licensing requirements](minimum-requirements.md#licensing-requirements).
 
+### Known issues and limitations
+
+Only Microsoft Edge is supported for web filtering in windows 10 multisession.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-windows-multi-session-device.md]:
```suggestion
Only Microsoft Edge is supported for web filtering in Windows 10 multi-session.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**